libraries/resource_providers/image_streamer/api800.rb

```diff
@@ -0,0 +1,37 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require_relative 'api600'
+
+module OneviewCookbook
+  module ImageStreamer
+    # Module for Image Streamer API 800 Resources
+    module API800
+      # Get resource class that matches the type given
+      # @param [String] type Name of the desired class type
+      # @param [String] variant Variant is ignored
+      # @return [Class] Resource class or nil if not found
+      def self.provider_named(type, _variant = nil)
```

Comment:
There should be a class inside the module and later function(def) inside the class. Is class missing here?

Reply:
Thats the structure is being followed in all other modules too.